scripts/installOpenBLAS.ps1

```diff
@@ -1,17 +1,25 @@
 Write-Host 'script installOpenBLAS.ps1 started'
-$VerbosePreference = "Continue" # display verbose messages
 New-Item -Path 'C:\BLAS' -ItemType Directory -Force # create directory
 # Enforce stronger cryptography
 [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.SecurityProtocolType]::Tls12
-$uri = 'https://sourceforge.net/projects/openblas/files/v0.3.6/OpenBLAS-0.3.6-x64.zip/download'
-$output = 'C:\BLAS\OpenBLAS-0.3.6-x64.zip'
-# Invoke-WebRequest $uri -OutFile $output
+$uri = 'https://github.com/xianyi/OpenBLAS/archive/v0.3.10.zip'
+$output = 'C:\BLAS\v0.3.10.zip'
 $webclient = New-Object System.Net.WebClient
 $webclient.DownloadFile($uri,"$output")
-Expand-Archive -Path 'C:\BLAS\OpenBLAS-0.3.6-x64.zip' -DestinationPath 'C:\BLAS\OpenBLAS-0.3.6-x64' -Force # expand zip file
-New-Item -Path Env:BLAS_LIBS -Value "/LIBPATH:C:\BLAS\OpenBLAS-0.3.6-x64\lib libopenblas.lib" -Force # create environment variable
-New-Item -Path Env:BLAS_CFLAGS -Value "/IC:\BLAS\OpenBLAS-0.3.6-x64\include" -Force # create environment variable
-Get-ChildItem 'C:\BLAS\OpenBLAS-0.3.6-x64' -Recurse # check for files
+Expand-Archive -Path 'C:\BLAS\v0.3.10.zip' -DestinationPath 'C:\BLAS\OpenBLAS-v0.3.10' -Force # expand zip file
+C:\Users\travis\build\AMICI\scripts\compileBLAS.cmd
+New-Item -Path 'C:\BLAS\lib' -ItemType Directory -Force # create directory
+Copy-Item "C:\BLAS\OpenBLAS-v0.3.10\OpenBLAS-0.3.10\lib\Release\openblas.lib" -Destination "C:\BLAS\lib" -Recurse
+New-Item -Path 'C:\BLAS\bin' -ItemType Directory -Force # create directory
+Copy-Item "C:\BLAS\OpenBLAS-v0.3.10\OpenBLAS-0.3.10\lib\openblas.dll" -Destination "C:\BLAS\bin" -Recurse
+Get-ChildItem -Path "C:\BLAS" -Include "openblas.lib" -Recurse # check for file
+Get-ChildItem -Path "C:\BLAS" -Include "openblas.dll" -Recurse # check for file
+[System.Environment]::SetEnvironmentVariable("BLAS_LIBS", "/LIBPATH:C:\BLAS\lib openblas.lib", [System.EnvironmentVariableTarget]::Machine)
+[System.Environment]::SetEnvironmentVariable("BLAS_LIBS", "/LIBPATH:C:\BLAS\lib openblas.lib", [System.EnvironmentVariableTarget]::User)
+[System.Environment]::SetEnvironmentVariable("BLAS_LIBS", "/LIBPATH:C:\BLAS\lib openblas.lib", [System.EnvironmentVariableTarget]::Process)
+[System.Environment]::SetEnvironmentVariable("BLAS_CFLAGS", "-IC:\BLAS\OpenBLAS-v0.3.10\OpenBLAS-0.3.10", [System.EnvironmentVariableTarget]::Machine)
+[System.Environment]::SetEnvironmentVariable("BLAS_CFLAGS", "-IC:\BLAS\OpenBLAS-v0.3.10\OpenBLAS-0.3.10", [System.EnvironmentVariableTarget]::User)
+[System.Environment]::SetEnvironmentVariable("BLAS_CFLAGS", "-IC:\BLAS\OpenBLAS-v0.3.10\OpenBLAS-0.3.10", [System.EnvironmentVariableTarget]::Process)
```

Comment:
These 6 lines are not required for travis. @thomassligon : will that store those variables permanently? I guess the the ::Machine lines work only as admin user?!

Reply:
If they have any value for a regular user, we can leave them in. Otherwise they should be removed.

Reply:
Removing now. To be added later if relevant. Time to merge this monster.